src/views/map.c

```diff
@@ -19,6 +19,7 @@
 #include "common/collection.h"
 #include "common/darktable.h"
 #include "common/debug.h"
+#include "common/gpx.c"
```

Comment:
use gpx.h instead of gpx.c

Reply:
If I move the two geodesic functions definitions into gpx.h, the compiler stops saying:

```
/home/git/darktable/src/common/gpx.h:90:13: error: ‘dt_gpx_geodesic_intermediate_point’ defined but not used [-Werror=unused-function]
   90 | static void dt_gpx_geodesic_intermediate_point(const double lat1, const double lon1,
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.
cc1: all warnings being treated as errors

```